apps/desktop/e2e/tests/project-tabs.spec.ts

```
@@ -0,0 +1,129 @@
+
+    // Set up project paths
+    projectPath1 = path.join(fixture.tempDir, 'project');
+    projectPath2 = createSecondProject(fixture);
+  });
+
+  test.afterAll(async () => {
+    await closeApp(fixture);
+  });
+
+  // eslint-disable-next-line no-empty-pattern
+  test.afterEach(async ({}, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)